lib/cb/clients/base.rb

```diff
@@ -14,6 +14,14 @@ class Base
       def self.cb_client
         @cb_client ||= Cb::Utils::Api.instance
       end
+
+      def self.headers(args)
+        {
+            'Accept' => 'application/json',
+            'Authorization' => "Bearer #{ args[:oauth_token] }",
```

Comment:
Also I want us to stop using the provided token.get methods as I don't want a dependency on that oauth wrapper.  Token management will be handled by the person calling the client.